cmd/arc/services/metamorph.go

```diff
@@ -280,7 +280,6 @@ func StartMetamorph(logger *slog.Logger) (func(), error) {
 			logger.Error("failed to shutdown mqClient", slog.String("err", err.Error()))
 		}
 
-		metamorphProcessor.Shutdown()
```

Comment:
What is the rationale behind removing the shutting down of the processor?

Reply:
It's called inside `server.Shutdown()`

Reply:
Yes, makes sense 👍